src/ResourceManager/TrafficManager/Commands.TrafficManager2/Models/TrafficManagerProfile.cs

```diff
@@ -41,37 +41,39 @@ public class TrafficManagerProfile
 
         public string MonitorPath { get; set; }
 
+        public int? MonitorIntervalInSeconds { get; set; }
+
+        public int? MonitorTimeoutInSeconds { get; set; }
+
+        public int? MonitorToleratedNumberOfFailures { get; set; }
+
         public List<TrafficManagerEndpoint> Endpoints { get; set; }
 
         public Profile ToSDKProfile()
         {
-            var profile = new Profile
-            {
-                Id = this.Id,
-                Name = this.Name,
-                Type = Constants.ProfileType,
-                Location = TrafficManagerClient.ProfileResourceLocation,
-                Properties = new ProfileProperties
+            var profile = new Profile(this.Id, this.Name, Constants.ProfileType, TrafficManagerClient.ProfileResourceLocation)
```

Comment:
I can, but it would just be duplication - the named parameters have the same names: Id, Name, Type, Location.